crates/bevy_reflect/src/impls/std.rs

```diff
@@ -201,6 +201,8 @@ impl_reflect_value!(::core::num::NonZeroI8(
     Serialize,
     Deserialize
 ));
+impl_reflect_value!(::core::num::Wrapping<T: Clone + Send + Sync>());
+impl_reflect_value!(::core::num::Saturating<T: Clone + Send + Sync>());
```

Comment:
Nit: It might be nice to add a test for these since derives on generics can be a bit wonky haha. Not required though since I believe other reflect_value generics are tested

Reply:
You can use `static_assertions` similar to how it was used #11407. Check out the [`assert_impl_all`](https://docs.rs/static_assertions/latest/static_assertions/macro.assert_impl_all.html) macro.